samples/appserver-sso-samples/sso-quickstart/src/main/java/org/wso2/appserver/samples/sso/Quickstart.java

```diff
@@ -0,0 +1,367 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.wso2.appserver.samples.sso;
+
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+
+import java.io.BufferedOutputStream;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.Socket;
+import java.net.URL;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+
+/**
+ * Quick start sample for sso.
+ */
+public class Quickstart {
+    private static final Log log;
+
+    static {
+        System.setProperty("org.apache.juli.formatter", "org.apache.juli.VerbatimFormatter");
+        log = LogFactory.getLog(Quickstart.class);
+    }
+
+    private Path wso2asPath = Paths.get("..", "..");
+    private Path wso2isPath = Paths.get("packs", "wso2is-5.1.0");
```

Comment:
We better make "wso2is-5.1.0" a constant, it's used several times in the java class and it is easier if we are to change the IS version later.